### PR TITLE
added primary swatch to theme

### LINKED ISSUE #25: Add a functionality to change the position of buttons & Joystick.
## Idea

We can add functionality to change the position of buttons & Joystick so the user can change the layout of remote according to their need.

### LINKED ISSUE #30: Add PrimarySwatch
**Note:** This issue is reserved for SLoP Contributors.
Refer: https://github.com/sinix-dev/docs/blob/master/tailwind.config.js

### PR DESCRIPTION
**Pull request by GajjarMiten**
## Description

added primary swatch to the theme of the application.

## Related Issues

Fixes  #30

### CODE REVIEW

File: lib/helpers/theme.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+final ThemeData theme = ThemeData(
+  fontFamily: 'Roboto',
+  primarySwatch: createMaterialColor(Color(0xffF33D61)),
```
Comment (sanket143) [inline lib/helpers/theme.dart]:
```suggestion
  primarySwatch: createMaterialColor(Color(0xFFDC143C)),
```

Comment (sanket143) [reply lib/helpers/theme.dart]: should it be `#DC143C`?

**Comment by GajjarMiten** [comment]
@sanket143  I have made changes.

File: lib/main.dart [hunk 2]
```diff
@@ -29,7 +30,7 @@ class App extends StatelessWidget {
   Widget build(BuildContext context) {
     return GetMaterialApp(
       home: MaterialApp(
-        theme: ThemeData(fontFamily: 'Roboto'),
+        theme: theme,
```
Comment (sanket143) [inline lib/main.dart]: Add `theme` to `GetMaterialApp` instead of `MaterialApp`

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: lib/main.dart (hunk 2 above)
Comment (GajjarMiten) [reply lib/main.dart]: @sanket143, I have  updated changes.

**Comment by sanket143** [comment]
give @GajjarMiten 25 points

**Pull request merged**